guides/5)other_tutorials/plot_component_for_maps.md

```diff
@@ -0,0 +1,165 @@
+
+## Overview 
+    
+We will be using the New York City Airbnb dataset, which is hosted on kaggle [here](https://www.kaggle.com/datasets/dgomonov/new-york-city-airbnb-open-data).
+Using this data we will plot Airbnb locations on a map output and allow filtering based on price and location. Below is the demo that we will be buiding. ⚡️
+
+<gradio-app space="dawood/NYC-Airbnb-Map"> </gradio-app>
+
+
+## Step 1 - Loading CSV data 💾
+
+Let's start by loading the Airbnb NYC data from the csv data we [downloaded](https://www.kaggle.com/datasets/dgomonov/new-york-city-airbnb-open-data).
+
+```python
+import pandas as pd
+
+df = pd.read_csv("AB_NYC_2019.csv")
+
+def filter_map(min_price, max_price, boroughs):
+    new_df = df[(df['neighbourhood_group'].isin(boroughs)) & 
+            (df['price'] > min_price) & (df['price'] < max_price)]
+    names = df["name"].tolist()
+    prices = df["price"].tolist()
+    text_list = [(names[i], prices[i]) for i in range(0, len(names))]
+```
+
+In the code above, we first load the csv data into a pandas dataframe. Let's begin by defining a function that we will use as the prediction fuction for the gradio app. This function will accept the minimum price and maximum price range as well as the list of boroughs to filter the resulting map. We can use the passed in values (`min_price`, `max_price`, and list of `boroughs`) to filter the dataframe and create `new_df`. Next we will create `text_list` of the names and prices of each Airbnb to use as labels on the map.
+ 
+## Step 2 - Map Figure 🌐
+
+Plotly makes it easy to work with maps. Let's take a look below how we can create a map figure.
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure(go.Scattermapbox(
+            customdata=text_list,
+            lat=new_df['latitude'].tolist(),
+            lon=new_df['longitude'].tolist(),
+            mode='markers',
+            marker=go.scattermapbox.Marker(
+                size=6
+            ),
+            hoverinfo="text",
+            hovertemplate='<b>Name</b>: %{customdata[0]}<br><b>Price</b>: $%{customdata[1]}'
+        ))
+
+fig.update_layout(
+    mapbox_style="open-street-map",
+    hovermode='closest',
+    mapbox=dict(
+        bearing=0,
+        center=go.layout.mapbox.Center(
+            lat=40.67,
+            lon=-73.90
+        ),
+        pitch=0,
+        zoom=9
+    ),
+)
+```
+
+Above, we create a scatter plot on mapbox by passing it our list of latitudes and longitudes to plot markers.  We also pass in our custom data of names and prices for additional info to appear on every marker we hover over. Next we use `update_layout` to specify other map settings such as zoom, and centering.
+
+More info [here](https://plotly.com/python/scattermapbox/) on scatter plots using Mapbox and Plotly.
```

Comment:
It would be good to describe what other map types besides `Scattermapbox` are supported.